oak_channel/src/message.rs

```diff
@@ -96,6 +101,10 @@ impl Message for ResponseMessage {
         LENGTH_SIZE + INVOCATION_ID_SIZE + self.body.len()
     }
 
+    fn is_empty(&self) -> bool {
+        false
+    }
+
```

Comment:
clippy linted this mod since it's now public, and demanded I add this

https://rust-lang.github.io/rust-clippy/master/#/len_without_is_empty

Reply:
relented, and disabled this lint instead.